src/core/expression/qgsexpression.cpp

```diff
@@ -859,6 +859,7 @@ void QgsExpression::initVariableHelp()
   sVariableHelpTexts()->insert( QStringLiteral( "symbol_label" ), QCoreApplication::translate( "symbol_label", "Label for the symbol (either a user defined label or the default autogenerated label)." ) );
   sVariableHelpTexts()->insert( QStringLiteral( "symbol_id" ), QCoreApplication::translate( "symbol_id", "Internal ID of the symbol." ) );
   sVariableHelpTexts()->insert( QStringLiteral( "symbol_count" ), QCoreApplication::translate( "symbol_count", "Total number of features represented by the symbol." ) );
+  sVariableHelpTexts()->insert( QStringLiteral( "symbol_expression" ), QCoreApplication::translate( "symbol_expression", "Expression representing the given symbol." ) );
```

Comment:
"Symbol expression" is very vague to me. How do you feel about something more explicit like "legend_item_expression" ?

Reply:
I'm not sure how much of an improvement that is to be honest but I don't mind changing it if you think that's more appropriate.